src/styles/_index.scss

```
@@ -2,6 +2,7 @@
 @use './colors' as colors;
 @use './v2/variables' as vars-v2;
 @use './v2/colors' as colors-v2;
+@use './v2/sing-colors' as sing-colors;
```

scss変数使ってないので、もしかしたらこれいらないかも？

---

@Hiroshiba 
こちらは無いと読み込まれなさそうです…！
index.scssがおそらくエントリーポイント

---

あっ　そりゃそうですね。。。

じゃ、じゃあ、`as sing-colors`が要らないかも･･･とか･･･？
```suggestion
@use './v2/sing-colors';
```

---

@Hiroshiba 
こちら修正いたしましたー